tests/aws/services/es/test_es.py

```diff
@@ -66,6 +66,8 @@ def try_cluster_health(cluster_url: str):
     ], "expected cluster state to be in a valid state"
 
 
+# flaky run: https://github.com/localstack/localstack/actions/runs/9450891131/job/26030567990
```

Comment:
nit: I don't think it's necessary to have this link in the code, since the logs are not available anymore after a few weeks. The association to the PR with a description linking / explaining the runs is good enough I would say.

Reply:
removed 🧹